src/main/java/laddergame/view/InputView.java

```diff
@@ -0,0 +1,40 @@
+package laddergame.view;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.List;
+
+public class InputView {
+
+    private static final String NAME_SEPARATOR = ",";
+    private static final BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
scanner 대신 BufferedReader를 사용한 이유가 있을까요?

Reply:
콘솔 게임에서 입력이 느리다면 사용자가 답답하지 않을까 해서
 scanner보다 속도가 빠른 BufferedReader를 사용해봤어요!

Reply:
네 좋습니다 ㅎㅎ 컨벤션 상 아래가 맞겠네요
```suggestion
    private static final BufferedReader BUFFERED_READER = new BufferedReader(new InputStreamReader(System.in));
```